Source/WebCore/css/parser/CSSPropertyParserConsumer+Anchor.cpp

```diff
@@ -316,7 +362,7 @@ RefPtr<CSSValue> consumePositionArea(CSSParserTokenRange& range, CSS::PropertyPa
     }
     auto dim2 = *maybeDim2;
 
-    return valueForPositionArea(dim1, dim2);
+    return minimizePositionArea(dim1, dim2, PositionAreaContext::Specified);
```

Comment:
Specified and computed should use the same shortest serialization, we usually apply shortest serialization to both.

https://drafts.csswg.org/cssom/#serialize-a-css-value

Reply:
I had two different paths because it would require changes to the parsing test, which I don't want to do. But from your comment, I'd do it (minimize serialization for both specified and computed value)

Reply:
The spec is actually not entirely clear on this point, so I suggest we land this and then deal with the specified value serialization question later.

Reply:
Resolved to land this.